specification/iotcentral/resource-manager/Microsoft.IoTCentral/stable/2018-09-01/examples/Apps_CreateOrUpdate.json

```diff
@@ -67,6 +67,12 @@
       }
     },
     "202": {},
+    "400": {
```

Comment:
This change should not be necessary - 400 is already covered by the `default` case below, which uses the same error contract. What's the motivation for the change?